packages/block-editor/src/components/editable-text/index.js

```diff
@@ -9,7 +9,14 @@ import { forwardRef } from '@wordpress/element';
 import RichText from '../rich-text';
 
 const EditableText = forwardRef( ( props, ref ) => {
-	return <RichText ref={ ref } { ...props } __unstableDisableFormats />;
+	return (
+		<RichText
+			ref={ ref }
+			{ ...props }
+			__unstableDisableFormats
+			preserveWhiteSpace
```

Comment:
What's the purpose of this PR. Should it be enabled automatically for all EditableText components?

Reply:
Yes, I think, because in any kind of plain text, white space should not be collapsed (like a text area).